packages/hierarchy-browser/src/common/dataProviders/CommunityDataProvider.ts

```diff
@@ -157,11 +156,13 @@ export class CommunityDataProvider {
 		}
 	}
 
+	// <--- Load Neighbor Community Data --->
```

Comment:
Can you use jsdoc format for public methods? This is extra important for anything client-facing.